doc/tutorials/app/orbbec_uvc.markdown

```diff
@@ -0,0 +1,147 @@
+Using Orbbec 3D cameras (UVC) {#tutorial_orbbec_uvc} 
+======================================================
+
+@tableofcontents
+
+@prev_tutorial{tutorial_orbbec_astra_openni}
+@next_tutorial{tutorial_intelperc}
+
+
+### Introduction
+
+This tutorial is devoted to the Orbbec 3D cameras based on UVC protocol. For the use of the older Orbbec 3D which depends on OpenNI, please refer to the [previous tutorial](https://github.com/opencv/opencv/blob/4.x/doc/tutorials/app/orbbec_astra_openni.markdown).
+
+Unlike working with the OpenNI based Astra 3D cameras which requires OpenCV built with OpenNI2 SDK, Orbbec SDK is not required to be installed for accessing Orbbec UVC 3D cameras via OpenCV. By using `cv::VideoCapture` class, users get the stream data from 3D cameras, similar to working with USB cameras. The calibration and alignment of the depth map and color image are done internally.
+
+
+
```

Comment:
Please remove extra empty lines here and below in the document.

Reply:
Thanks! I have fixed the two problems you mentioned above.